DOCSUP-14537: Document the log_queries_probability setting (#29099)
ClickHouse/ClickHouse: docs/en/operations/system-tables/query_views_log.md

ClickHouse does not delete data from the table automatically. See [Introduction](../../operations/system-tables/index.md#system-tables-introduction) for more details.

> You can use the [log_queries_probability](../../operations/settings/settings.md#log-queries-probability) setting to set the probability with which queries are registered in the `query_views_log` table.

Reviewer: ```suggestion
You can use the [log_queries_probability](../../operations/settings/settings.md#log-queries-probability) setting to reduce the number of queries, registered in the `query_views_log` table.
```
